data/data-pipeline/README.md

```diff
@@ -6,30 +6,30 @@ We use Docker to install the necessary libraries in a container that can be run
-- Generate Score: `docker run --rm -it j40_score /bin/sh -c "python3 application.py score-run"`
+Here's a list of commands:
 
-## Log visualization
-
-If you want to visualize logs while running a command, the following temporary workaround can be used:
-
-- Run `docker-compose up` on the root of the repo
-- Open a new tab on your terminal
-- Then run any command for the application using this format: `docker exec j40_score_1 python3 application.py [command]`
+- Get help: `docker exec j40_data_pipeline_1 python3 application.py --help"`
+- Clean up the census data directories: `docker exec j40_data_pipeline_1 python3 application.py census-cleanup"`
+- Clean up the data directories: `docker exec j40_data_pipeline_1 python3 application.py data-cleanup"`
+- Generate census data: `docker exec j40_data_pipeline_1 python3 application.py census-data-download"`
+- Run all ETL processes: `docker exec j40_data_pipeline_1 python3 application.py etl-run"`
+- Generate Score: `docker exec j40_data_pipeline_1 python3 application.py score-run"`
 
 ## Local development
 
-You can run the Python code locally without Docker to develop, using Poetry. However, to generate the census data you will need the [GDAL library](https://github.com/OSGeo/gdal) installed locally. Also to generate tiles for a local map, you will need [Mapbox tippeanoe](https://github.com/mapbox/tippecanoe)
+You can run the Python code locally without Docker to develop, using Poetry. However, to generate the census data you will need the [GDAL library](https://github.com/OSGeo/gdal) installed locally. Also to generate tiles for a local map, you will need [Mapbox tippeanoe](https://github.com/mapbox/tippecanoe). Please refer to the repos for specific instructions for your OS.
+
+### Windows Users
+- If you want to download Census data or run tile generation, please install TippeCanoe [following these instrcutions](https://github.com/GISupportICRC/ArcGIS2Mapbox#installing-tippecanoe-on-windows).
+- If you want to generate tiles, you need some pre-requisites for Geopandas as specified in the Poetry requirements. Please follow [these instructions](https://stackoverflow.com/questions/56958421/pip-install-geopandas-on-windows) to install the Geopandas dependency locally.
 
-Note: If you are using Windows, please follow [these instructions](https://stackoverflow.com/questions/56958421/pip-install-geopandas-on-windows) to install Geopandas locally. If you want to install TippeCanoe, [follow these instrcutions](https://github.com/GISupportICRC/ArcGIS2Mapbox#installing-tippecanoe-on-windows).
+### Setting up Poetry
 
 - Start a terminal
+- Change to this directory (`/data/data-pipeline`)
 - Make sure you have Python 3.9 installed: `python -V` or `python3 -V`
 - We use [Poetry](https://python-poetry.org/) for managing dependencies and building the application. Please follow the instructions on their site to download.
 - Install Poetry requirements with `poetry install`
```

Comment:
There are a couple of lines under this that need updating (`cd score` should be `cd data-pipeline` now I think): lines 41, 53, 59, 64

Reply:
Good catch! Fixed